src/mmrelay/migrate.py

```
@@ -0,0 +1,2056 @@
+            "new_path": str(new_creds),
+            "action": "move" if move else "copy",
+            "dry_run": True,
+        }
+
+    if new_creds.exists() and not force:
+        logger.info("Backing up existing credentials: %s", new_creds)
+        backup_path = _backup_file(new_creds)
+        try:
+            shutil.copy2(str(new_creds), str(backup_path))
+        except (OSError, IOError) as e:
+            logger.exception("Failed to backup credentials: %s", e)
```

Ruff (0.14.14)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)